PRFCS/prfc-1.md

```diff
@@ -51,50 +55,50 @@ Returns information about the Token implemented by this contract.
 fn allowance(owner: PublicAddress, spender: PublicAddress) -> u64
 ```
 
-Returns the number of tokens currently in the spender's allowance that they can spend on the ‘owner’s’ behalf.
+Returns the amount of tokens currently in the spender's allowance that they can spend on the owner’s behalf.
 
 ### balance_of
 ```rust
 fn balance_of(address: PublicAddress) -> u64
 ```
 
-Queries the balance for an owner account 'address'.
+Queries the balance for an owner account `address`.
 
 
 ## Required Calls
 ---
 
 ### transfer
 ```rust
-fn transfer(to_address: Option<Public_Address>, amount: u64)
+fn transfer(to_address: Option<PublicAddress>, amount: u64)
 ```
 
-`transfer` transfers tokens to an address from the account identified by `txn.signer`. If `to_address` is None, this burns the amount.
+`transfer` transfers tokens to an address from the account identified by `calling_account`. If `to_address` is None, this burns the amount.
```

Comment:
Can directly start with "Transfers tokens to an address..", we don't have to repeat the method name. 
Same applies on other required calls to match with the format in Required Views and PRFC2.